code/game/objects/items/mountable_frames/newscaster_frame.dm

```diff
@@ -4,13 +4,51 @@
 	icon_state = "newscaster"
 	item_state = "syringe_kit"
 
-	materials = list(MAT_METAL=14000, MAT_GLASS=8000)
+	materials = list(MAT_METAL=6000, MAT_GLASS=2000)
 	mount_requirements = MOUNTED_FRAME_SIMFLOOR | MOUNTED_FRAME_NOSPACE
-	metal_sheets_refunded = 7
-	glass_sheets_refunded = 4
+	metal_sheets_refunded = 3
+	glass_sheets_refunded = 1
 
 /obj/item/mounted/frame/newscaster_frame/do_build(turf/on_wall, mob/user)
 	var/obj/machinery/newscaster/N = new /obj/machinery/newscaster(get_turf(src), get_dir(on_wall, user), 1)
 	N.pixel_y -= (loc.y - on_wall.y) * 32
 	N.pixel_x -= (loc.x - on_wall.x) * 32
 	qdel(src)
+/obj/item/mounted/frame/newscaster_frame/display_frame
+	name = "status display frame"
+	desc = "Used to build status displays, just secure to the wall."
+	icon = 'icons/obj/status_display.dmi'
+	icon_state = "frame"
+	item_state = "syringe_kit"
+
+/obj/item/mounted/frame/newscaster_frame/display_frame/do_build(turf/on_wall, mob/user)
```

Comment:
I think instead of copy pasting the code. It would be better to have a `build_type` variable on `/obj/item/mounted/frame/newscaster_frame` which defines which object to actually create.

It might also be better to make an abstract parent for all these build paths. So they don't inherit from `/obj/item/mounted/frame/newscaster_frame` but instead from `/obj/item/mounted/frame/display` or something the like. And `/obj/item/mounted/frame/newscaster_frame` will become `/obj/item/mounted/frame/display/newscaster_frame`

Reply:
done